### ISSUE TITLE
Symfony's WebDebug Toolbar overlaps the lower Victoire Bar

### ISSUE THREAD

**Issue by mablae** [Victoire/victoire#873]
## Description
The Victoire CMS Bar in inaccessible in app_dev.php

## Resolution Ideas
Introduce a margin-bottom and display 2 rows for the 2 toolbars

## Related to
UI Bundle?

**Comment by paulandrieux** [comment]
You are right, it would be perfect to apply margin only when user is logged-in with ROLE_VICTOIRE
